### PR TITLE
feat: Added support for plural_name for foods

### PR DESCRIPTION
**Pull request by Choromanski**
## What type of PR is this?

- feature

## What this PR does / why we need it:

This PR allows the seeder to import plural names for foods.
This allows for new user have cleaner ingredient lists without having to do any additional configuration.

## Which issue(s) this PR fixes:

N/A

## Special notes for your reviewer:

1. This work was discussed in #3933 
2. If anything is done that causes issues with crowdin let me know and I can work to resolve it.
3. I only provided plurals for foods that would normally be the unit (ex. 1 watermelon)
4. I sorted the list by keys so that I could de-duplicate easier
5. I also removed some words that were duplicated in there UK equivalent (ex. aubergine/eggplant, broad/fava beans, icing/powdered sugar)
6. Any comments or recommendations would be greatly appreciated

## Testing

Ran a fresh install and all of the plural ingredients were seeded properly:
image

### CODE REVIEW

**Comment by michael-genson** [comment]
@hay-kot would you be able to look at this one? Just want to make sure it won't break existing translations (or if it will, we have some way of fixing them)

**Comment by hay-kot** [comment]
I believe that Crowdin will automatically pick up the changes, however the other files would get overridden, so I think this PR also needs to migrate all the other language files as well.

**Comment by Choromanski** [comment]
I will work on updating the other files

**Comment by Choromanski** [comment]
@hay-kot if you would like to pull the update from crowdin again let me know and I can reformat with the new updats

File: mealie/repos/seed/resources/foods/locales/es-ES.json [hunk 1]
```diff
@@ -1,222 +1,692 @@
...
+    },
+    "cumin": {
+        "name": "comino"
+    },
+    "daikon": {
+        "name": "daikon",
+        "plural_name": "daikons"
+    },
+    "dairy-products-and-dairy-substitutes": {
+        "name": "productos l\u00e1cteos y sustitutos de l\u00e1cteos"
```
Comment (hay-kot) [inline mealie/repos/seed/resources/foods/locales/es-ES.json]: FYI, looks like there's encoding issues from however you transformed these.

Comment (Choromanski) [reply mealie/repos/seed/resources/foods/locales/es-ES.json]: This should be resolved now

**Comment by Choromanski** [comment]
Whoops, I destroyed this PR, will create a new one

**Pull request closed without merging**